### PR TITLE
more dev notes for running locally

### PR DESCRIPTION
**Pull request by Flet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,19 @@
...
+- `secret\index.js` will need to be created locally (copy `secret\index-sample.js`).
 
-The following folders are required to exist with chmod `777`.
+### Log folders
 
+For production, the following folders are required to exist with chmod `777`.
 - $HOME/www/log/webtorrent.io/crash-reports
 - $HOME/www/log/webtorrent.io/telemetry
 
+For local development (not using `--production` flag), these folders will be created at
```
Comment (dcposch) [inline README.md]: not using the ... flag

Comment (dcposch) [inline README.md]: np

**Comment by dcposch** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (feross) [inline README.md]: The way to put the server into production mode is actually `NODE_ENV="production"`, like most node server apps, not with a `--production` flag. I'll update this.

Comment (Flet) [inline README.md]: Oops. thanks :)

Comment (feross) [inline README.md]: No problem :)

**Pull request merged**